app/javascript/.storybook/preview-head.html

```diff
@@ -3,5 +3,8 @@
     getOSKeyboardModifierKeyString: function () {
       return 'cmd';
     },
+    isNativeIOS: function () {
```

Comment:
as this check happens in the ImageUploader, we need to stub this in storybook so we can continue to show the toolbar component there